FAQ.md

```diff
@@ -381,3 +382,11 @@ it can [cause
 problems](https://groups.google.com/d/msg/golang-nuts/AnMr9NL6dtc/UnyUUKcMCAAJ).
 If your dependers are using `dep`, this is not a concern, as `dep` takes care of
 stripping out nested `vendor` directories.
+
+## How do I configure a dependency that doesn't tag its releases?
+
+Add a constraint to your manifest that specifies `"branch": "master"`
```

Comment:
Ooops, I missed this the first time around but it should be `branch = "master"` in order to be proper TOML.